modules/core/src/main/java/org/apache/ignite/ssl/SslContextFactory.java

```diff
@@ -89,6 +90,12 @@
     /** Trust managers. */
     private TrustManager[] trustMgrs;
 
+    /** Enabled cipher suites. */
+    private String[] cipherSuites;
+
+    /** Enabled cipher suites. */
```

Comment:
We already have sslContextFactory in igniteConfiguration which has setProtocol().
Having protocols here is redundant and it is not obvious which one of two will be used. I suggest dropping it, using new String[] {sslContextFactory.create().getProtocol()} in its place

Reply:
it's not the same, the protocol field allows to set only one particular
protocol, while protocols allow you to set several different versions, for
example with protocols you can set:
TLSv1.1 and TLSv1.2, but exclude TLSv1.0